lib/radiator/directory/editor.ex

```diff
@@ -421,6 +423,44 @@ defmodule Radiator.Directory.Editor do
     episode
   end
 
+  def get_chapter(actor = %Auth.User{}, id) do
```

Comment:
I'd rather have us use the combination audio_id,start as primary keys here. of course multiple identical start values are forbidden then and need to be constraint against.